packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -56,6 +61,15 @@ class _DefaultDoctorValidatorsProvider implements DoctorValidatorsProvider {
   List<DoctorValidator> _validators;
   List<Workflow> _workflows;
 
+  final extension.ExtensionShim extensionShim = extension.ExtensionShim(<extension.ToolExtension>[
```

Comment:
This is a bit of a temporary measure until we can bootstrap something that works in g3 too.